packages/manager/src/components/CheckoutSummary/SummaryItem.tsx

```diff
@@ -1,16 +1,19 @@
 import { styled } from '@mui/material/styles';
+import { useTheme } from '@mui/styles';
 import React from 'react';
 
 import { Grid } from '../Grid';
 import { Typography } from '../Typography';
 import { SummaryItem as Props } from './CheckoutSummary';
 
 export const SummaryItem = ({ details, title }: Props) => {
+  const theme = useTheme();
+
   return (
     <StyledGrid item>
       {title ? (
         <>
-          <Typography component="span" sx={{ fontWeight: 'bold' }}>
+          <Typography component="span" fontFamily={theme.font.bold}>
```

Comment:
We could also convert this to 
```suggestion
          <Typography component="span" sx={(theme) => ({ fontFamily: theme.font.bold })}>
```

if we want to avoid importing the useTheme? Same with for `DynamicPricingNotice` and `SelectRegionPanel`.

I guess in general, one of my questions is should we do this type of styling or using sx?